packages/Snackbar/README.md

```diff
@@ -159,7 +159,3 @@ To add some action to the Snackbar, just add options about the action to the sho
 ```
 
 ![Feb-25-2020 00-16-47](https://user-images.githubusercontent.com/17980230/75164429-265a0000-5764-11ea-9c6f-12bf362dc32b.gif)
-
-- More Complex Examples
-
-You can find more complex usages on this [storybook codes](storybook/stories/Snackbr.stories.tsx). 
```

Comment:
Thank you for your interest. 
As the directory structure changed, the file was moved to 'stories/packages/snackbar.stories.tsx'. 
I think it would be better to change this line to the changed location rather than delete it, what do you think?

Reply:
Your suggestion would be like better. Then, how about adding snackbar doc to 'doc/packages' than stories.tsx file.

Reply:
I didn't understand your proposal. You mean that make another document file to 'doc/packages'? Then I think that's good.